android/src/main/java/com/staltz/react/workers/WorkersManager.java

```diff
@@ -83,20 +95,17 @@ public void run() {
 
   @ReactMethod
   public void stopWorker(final Integer key) {
-    final WorkersInstance worker = this.workers.remove(key);
-    Assertions.assertNotNull(worker);
-
-    new Handler(Looper.getMainLooper()).post(new Runnable() {
-      @Override
-      public void run() {
-        worker.stop();
-      }
-    });
+    this.workers.remove(key);
```

Comment:
why remove the `worker.stop()` here?

Reply:
There's a bug in this lib with the combination with RNN, if you stop it, it destroys the ReactInstanceManger so we just create a new worker when the user comes back rather than reusing the same.